tests/unit/style/parsing/image.rs

```diff
@@ -32,6 +38,12 @@ fn test_linear_gradient() {
 
     // Parsing without <angle> and <side-or-corner>
     assert_roundtrip_with_context!(Image::parse, "linear-gradient(red, green)");
+
+    // AngleOrCorner::None should become AngleOrCorner::Angle(Angle(PI)) when parsed
```

Comment:
Maybe add a link to the spec in the comment. Or at least make clear that `PI` means "top-to-bottom". :)